.travis.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           - 'android-sdk-preview-license-52d11cd2'
           - 'android-sdk-license-.+'
           - 'google-gdk-license-.+'
-      before_install: nvm install 8
+      before_install: nvm install 10
```

Comment:
cc @y-lohse :D

Reply:
Ah yes, j'avais pas pensé a nvm qui peut s'installer [en bash](https://github.com/nvm-sh/nvm#installing-and-updating).Je vais mettre a jour sur drive.